sdk/resourcemanager/Azure.ResourceManager/README.md

```diff
@@ -73,31 +73,24 @@ It also has access to all of the operations and like the **[Resource]Operations*
 to a specific resource in Azure.
 
 ### Structured Resource Identifier
-Instead of implementing your own parsing logic, you can implicitly cast a resource identifier string into an object which will do the parsing for you.
-
-There are 3 types of ResourceIdentifiers and they correspond to which level the resource lives at:
-- A resource that lives on a tenant will have a `TenantResourceIdentifier`.
-- A resource that lives under a subscription will have a `SubscriptionResourceIdentifer`.  
-- A resource that lives under a resource group will have a `ResourceGroupResourceIdentifier`.
-
-You can usually tell by the id string itself which type it is, but if you are unsure you can always cast it onto a `ResourceIdentifier` and use the Try methods to retrieve the values.
+Resource IDs contains useful information about the resource itself, but they are plain strings that have to be parsed. Instead of implementing your own parsing logic, you can use a `ResourceIdentifier` object which will do the parsing for you: `new ResourceIdentifer("myid");`.
```

Comment:
```suggestion
Resource IDs contain useful information about the resource itself, but they are plain strings that have to be parsed. Instead of implementing your own parsing logic, you can use a `ResourceIdentifier` object which will do the parsing for you: `new ResourceIdentifer("myid");`.
```